public/doc/openapi-3-v1.0.0.json

```diff
@@ -564,6 +564,40 @@
         }
       }
     },
+    "/portfolios/{portfolio_id}/icon": {
```

Comment:
Was there no change needed to the /portfolio_items/{portfolio_item_id}/icon endpoint due to this change? I see the removal of the POST for portfolio_item on line 1605.

Reply:
Basically this changeset removes the ability for an `Icon` to be created without a corresponding `PortfolioItem` or `Portfolio` (no dangling references!). This negates the `POST /portfolio_items/{portfolio_item_id}/icon` which was just to add a specified icon to a portfolio_item, since now one would just create an icon while specifying the `portfolio_item` or `portfolio` id.

Reply:
After conversations around this endpoint offline. It has come to our attention that the openapi spec is missing the declaration of `:portfolio_id`, `:portfolio_item_id` params that either of are required when calling `POST` on `/icons`.